pennychat/views.py

```diff
@@ -16,8 +19,26 @@ class PennyChatViewSet(viewsets.ModelViewSet):
     serializer_class = PennyChatSerializer
 
     def perform_create(self, serializer):
-        # TODO add participants and make request user's UserProfile as the Organizer
-        serializer.save()
+        if not self.request.auth:
```

Comment:
It looks like this whole class is authenticated so we could use a permissions class.

Reply:
The list and detail are not for this one

Reply:
It might make sense to use a decorator then to encapsulate the logic

Reply:
That makes sense, I'll do that.